src/forge/actors/generator.py

```diff
@@ -92,6 +95,8 @@ class Generator(ForgeActor):
     engine_args: EngineArgs | Mapping = field(default_factory=EngineArgs)
     sampling_params: SamplingParams | Mapping = field(default_factory=SamplingParams)
     use_dcp_for_weight_sync: bool | None = None
+    prefetch_weights_to_shm: bool = True
```

Comment:
In general we should try to avoid changing the "public" api when we expect to quickly change the backend again. After launch we should try to keep this in mind.